torch/nn/modules/module.py

```diff
@@ -1492,6 +1496,8 @@ def _slow_forward(self, *input, **kwargs):
         return result
 
     def _call_impl(self, *args, **kwargs):
+        if self._compiled_call_impl is not None:
+            return self._compiled_call_impl(*args, **kwargs)
```

Comment:
This is this same function `_call_impl` right? How is this not infinite recursing? :p

Reply:
Ah because it was never called, just added this to my test `model(torch.rand(1, 10))` and the problem was found out
`

Reply:
Apparently not, looks like I broke all hooks lol